content/en/blog/2023/euwg-future-of-observability-panel/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title:
+  'OTel End User Working Group Presents: The Future of Observability Panel
+  Discussion'
+linkTitle: The Future of Observability Panel
+date: 2023-10-17
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep)
+cSpell:ignore: Dyrmishi Farfetch Iris Mellifera Nočnica Samuel Vijay youtube
+---
+
+![The Evolution of Observability Practices, featuring Guests David Wynn, Austin Parker, Vijay Samuel, Nočnica Mellifera, and Iris Dyrmishi](evolution-o11y-practices-social-card.jpeg)
```

Comment:
Just a suggestion, but maybe have the youtube video here already, so people can dive into it directly?

Reply:
Great idea @svrnm! I've added it just after the intro paragraph.